macos/FluentUITestViewControllers/TestLabelViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+
+class TestLabelViewController: NSViewController {
+
+	override func loadView() {
+
+
+		let labels = TextStyle.allCases.map {textStyle -> NSTextField in
+			return Label(withString: string(ForTextStyle: textStyle), forTextStyle: textStyle)
+		}
+		
+		let buttons = TextStyle.allCases.map {textStyle -> Button in
+			return Button(title: string(ForTextStyle: textStyle), textStyle: textStyle, style: .primaryFilled)
+		}
+		
+		let links = TextStyle.allCases.map {textStyle -> Link in
+			return Link(title: string(ForTextStyle: textStyle), textStyle: textStyle)
+		}
+	
+
+		let gridView = NSGridView(frame: .zero)
+		gridView.rowSpacing = gridViewRowSpacing
+		gridView.columnSpacing = gridViewColumnSpacing
+		gridView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+
+		gridView.addColumn(with: labels)
+		gridView.addColumn(with: buttons)
+		gridView.addColumn(with: links)
+
+		let containerView = NSStackView()
+		containerView.orientation = .vertical
+		containerView.translatesAutoresizingMaskIntoConstraints = false
+		containerView.edgeInsets = NSEdgeInsets(
+			top: containerViewEdgeInsets,
+			left: containerViewEdgeInsets,
+			bottom: containerViewEdgeInsets,
+			right: containerViewEdgeInsets
+		)
+
+		containerView.addView(gridView, in: .top)
+
+		view = containerView
+	}
+	
+	func string(ForTextStyle textStyle: TextStyle) -> String {
+		switch textStyle {
+
```

Comment:
extra newline (we don't have one after the switch above so let's be consistent)

Reply:
> I'm not sure I see the value in providing mappings for macOS text styles. People can and should just use them directly. As you mention, we should map to FURN (and also fluentui-ios). Adding these temporary ones in the interim seems like it's only indirection rather than values and then will make mapping to fluentui styles a behavioral and breaking change. Why not wait or do a best guess translation to fluent text styles to start with?

Only macOS Big Sur users and up can use these text syles. I wanted to add a style for the platforms we do support. Furthermore, I'm pretty sure that iOS does the same/similar thing of mimicking the system API. That being sad, If we don't want to introduce a breaking change later and just want to map to the FURN styles now, that's fine. I wanted to start the conversaton with keeping the system text styles in mind first, and this was the easiest way to do that.